File: source/demo/javascript/js/function/005.js
function max() {
	var max = Number.NEGATIVE_INFINITY;
	for(var i = 0; i < arguments.length; i++) {
		if(arguments[i] > max) max = arguments[i];
	}
	return max;
}

var largest = max(1, 10, 100, 2, 3, 1000, 4, 5, 10000, 6);
console.log(largest);

File: source/c/c primer plus/test/第六章/005.c
/**
 * ±à³ÌÁ·Ï°6.16 µÚ5Ìâ
 */
#include <stdio.h>
int main(void){
	int max, min, index;
	printf("Please enter the min number: ");
	scanf("%d", &min);
	printf("enter the max number: ");
	scanf("%d", &max);
	for(index = min; index <= max; index++){
		printf("%4d%8d%8d\n", index, index * index, index * index * index);
	}
	getch();
	return 0;
}
